docs/kpt_functions.md

```diff
@@ -0,0 +1,59 @@
+# Policy Library KPT Functions
+This repo contains several kpt functions to manage documentation and make it easier to pull in policy for Config Validator.
+
+These functions are meant to be run against this repo. Before running one of the below functions run `kpt get`:
+
+```
+kpt pkg get https://github.com/forseti-security/policy-library.git ./policy-library
+cd ./policy-library
+```
+
+## Generate Docs
+Generates markdown documentation for the templates and constraints in this repo.
+
+### Run with Node
+Generate the markdown docs.
+
+```
+kpt fn source ./samples/ ./policies/ | 
+node bundler/dist/generate_docs_run.js -d sink_dir=./docs -d overwrite=true
```

Comment:
We shouldn't require people to run with node (in fact, I wouldn't mention this in docs at all). We should highlight the `kpt fn run` verison (https://googlecontainertools.github.io/kpt/reference/fn/#examples).

Reply:
👍